app/backend/src/clj/teet/admin/admin_commands.clj

```diff
@@ -0,0 +1,20 @@
+(ns teet.admin.admin-commands
+  (:require [teet.db-api.core :as db-api]
+            [datomic.client.api :as d]
+            [teet.user.user-roles :as user-roles]
+            [taoensso.timbre :as log]))
+
+(defn- new-user []
+  {:user/id (java.util.UUID/randomUUID)
+   :user/roles [:user]})
+
+(defmethod db-api/command! :admin/create-user [{conn :conn} user-data]
+  (let [user (merge (new-user)
+                    (select-keys user-data [:user/id :user/person-id :user/roles]))]
+    (log/info "CREATE USER: " user)
+    (d/transact conn
+                {:tx-data [user]})
+    :ok))
+
+(defmethod db-api/command-authorization :admin/create-user [{user :user} _]
```

Comment:
I'd rather see the authorization closely tied to the command/query definitions than as multimethods. I guess we can have a convention of having the authorization defined next to the command/query definition, but I don't immediately see what additional value using multimethods adds here.

Reply:
Well that is an issue for another PR, the query/command authorization split is existing.